src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -44,15 +46,17 @@ class ConsumeMessagesCommand extends Command
     private $eventDispatcher;
     private $logger;
     private $receiverNames;
+    private $busIds;
     private $resetServicesListener;
 
-    public function __construct(RoutableMessageBus $routableBus, ContainerInterface $receiverLocator, EventDispatcherInterface $eventDispatcher, LoggerInterface $logger = null, array $receiverNames = [], ResetServicesListener $resetServicesListener = null)
+    public function __construct(RoutableMessageBus $routableBus, ContainerInterface $receiverLocator, EventDispatcherInterface $eventDispatcher, LoggerInterface $logger = null, array $receiverNames = [], array $busIds = [], ResetServicesListener $resetServicesListener = null)
```

Comment:
Every new argument must be put at end to not break BC.
```suggestion
    public function __construct(RoutableMessageBus $routableBus, ContainerInterface $receiverLocator, EventDispatcherInterface $eventDispatcher, LoggerInterface $logger = null, array $receiverNames = [], ResetServicesListener $resetServicesListener = null, array $busIds = [])
```

Reply:
This is what I did in a previous version, but I had errors with.
The new argument has been added for 5.4 (https://github.com/symfony/symfony/pull/43133), I think we can "break" compatibility since the code has not been released yet.

Reply:
Even if it's new, changing order of arguments means you need to adjust all the code that is already in repository.

Maybe better try something like this? https://github.com/symfony/symfony/pull/43891#discussion_r741896872